akka-actor/src/main/resources/reference.conf

```diff
@@ -1052,6 +1052,16 @@ akka {
       async-dns {
         provider-object = "akka.io.dns.internal.AsyncDnsProvider"
 
+        # To set upper bound the time to cache name resolutions
+        # Actually time to cache in successful case is lower value from positive-ttl and ttl from dns record
+        # In negativve case time to cache is negative-ttl value
+        # Possible values:
+        # forever: cache forever
+        # never: no caching
+        # n [time unit]: positive timeout with unit, for example "30 s"
+        positive-ttl = forever
+        negative-ttl = never
```

Comment:
I wonder if this would be better as this organization for config, allowing compartmentalizing the TTLs, 
and then you should IMO add a description, per config, e.g.
```
async-dns {

  # Overarching description
  # Possible values:
  # forever: cache forever
  # never: no caching
  ttl {
     # The description
     positive = forever
     # The description
     negative = never
  }
}
```

Reply:
Nice suggestion. I've tried to create a config like inet-address already has
```
 inet-address {
       provider-object = "akka.io.InetAddressDnsProvider"

        positive-ttl = default
        negative-ttl = default
  
        cache-cleanup-interval = 120s
      }
```
I think configs'll be a bit unclear If `AsyncDns` and `InetAddress`'ll have different config style. 
Should we worry about this? 
From the other point this is another config, in another config-block, and for setting another resolver.